src/igvfd/types/sample.py

```diff
@@ -107,6 +123,21 @@ class InVitroSystem(Biosample):
     item_type = 'in_vitro_system'
     schema = load_schema('igvfd:schemas/in_vitro_system.json')
 
+    @calculated_property(
+        schema={
+            'title': 'Summary',
+            'type': 'string',
+            'notSubmittable': True,
+        }
+    )
+    def summary(self, request, biosample_term, taxa, classification=None, time_post_factors_introduction=None, time_post_factors_introduction_units=None):
+        sample_term_object = request.embed(biosample_term, '@@object?skip_calculated=true')
+        term_name = sample_term_object.get('term_name')
+        if time_post_factors_introduction and time_post_factors_introduction_units:
+            return f'{term_name} {classification}, {taxa} ({time_post_factors_introduction} {time_post_factors_introduction_units})'
```

Comment:
Wondering if we should add more detail here about the time post factors introduction. Without additional context, my first assumption would be that the time refers to age.

Reply:
I think we can skip the verbs at this time
if and when labs complain we can consider adding verbs